notebooks/train_notebook.ipynb

```diff
@@ -1096,8 +1096,7 @@
       ],
       "source": [
         "from src.engine import evaluate\n",
-        "test = \"../model/nish_49m_deep6_cyclic_lr_bottleneck.pt\"\n",
-        "model.load_state_dict(torch.load(test))\n",
+        "model.load_state_dict(torch.load([file_path]))\n",
```

Comment:
Is the array wrapping the file path necessary? I got an error when I ran it on my desktop.

Reply:
but that might be related to having to run it on the CPU on my desktop as opposed to the GPU.